conf.yaml

```diff
@@ -5,6 +5,7 @@ meta:
 repository:
     owner: galaxyproject
     name: galaxy
+    next_milestone: "16.04"
```

Comment:
Iteration N+1 is automating this, but yes for now I think we can do it manually and add updating this value to the release checklist.